### ISSUE TITLE
funding leverages table cannot be edited

### ISSUE THREAD

**Issue by stefanprobst** [DARIAH-ERIC/dariah-unr#175]

**Comment by egray523** [comment]
Hello, I would like to call your attention to this open issue and try to push it to a higher priority - indeed, our Swedish colleagues mistakenly entered their amounts in their 2024 form ([direct link to page](https://dariah-unr.acdh-ch-dev.oeaw.ac.at/en/dashboard/reports/2024/countries/se/edit/project-funding-leverage)) in SEK, not in Euro, and would like to have them corrected. Here are teh correct figures:

> Huminfra period 1 “Total funding” should be 149,302 EUR
> 
> Huminfra period 2 “Total funding” should be 209,889 EUR
> 

1. As a first solution, with highest priority, someone with database access could correct the data.
2. A second, longer solution but better overall, would be to allow either for edits to be made via admin seciton of UNR, or 
3. Perhaps even allow changes/edits by NC accounts without having to pass by an admin. 

@babslgam @ArnaudRoi

**Comment by stefanprobst** [comment]
i updated the values in the db.

**Comment by laureD19** [comment]
cf. #157

**Comment by babslgam** [comment]
@stefanprobst have there been any discussions about how the editing workflow should look in the ui? (edit button per row or something like that?)

**Comment by stefanprobst** [comment]
no, but edit button per row makes sense - should open an edit dialog, which looks the same as the "Add project funding" dialog, with pre-populated fields and "Update" instead of "Create" as submit button and dialog title.

**Comment by stefanprobst** [comment]
editable via admin dashboard
